ocs_ci/templates/provider-client-deployment/provider_subscription.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openshift-storage
+    labels:
+    openshift.io/cluster-monitoring: "true"
+
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: openshift-storage
+    namespace: openshift-storage
+spec:
+    targetNamespaces:
+    - openshift-storage
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: odf-operator
+    namespace: openshift-storage
+spec:
+    channel: stable-4.15
```

Comment:
I think there should be a placeholder, if no we will be always forced to use odf 4.15.  Suggest to replace with '{}' even if this is an example